src/parse/asp/objects.go

```diff
@@ -967,6 +968,37 @@ func loadPluginConfig(state *core.BuildState, c pyDict) {
 	c[strings.ToUpper(pluginName)] = pluginNamespace
 }
 
+func toPyObject(key, val, toType string) pyObject {
+	if toType == "" || toType == "str" {
+		return pyString(val)
+	}
+
+	if val == "" {
+		return pyNone{}
```

Comment:
Should we do this? Or should they become the zero value (0 / False)?

Reply:
This is the difference between an optional boolean value that hasn't been set, and one that's explicitly been set to False. Perhaps we don't need to model that but it seems more correct.

Reply:
Yeah okay I'll change this.